Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart for SUSE Cloud Foundry
+name: cf
```

Comment:
I've pinged Troy, he is currently busy with other stuff.

I've created a separate ticket for it: https://trello.com/c/6E77Oadd/213-provide-metadata-for-the-chartyaml-files-for-scf-and-uaa